arabic.txt

```diff
@@ -3,5 +3,7 @@
 egydead.com#$#abort-on-property-read _pop; abort-on-property-read atob; abort-on-property-write Fingerprint2
 cimaclub.com#$#abort-on-property-read atob; abort-on-property-read decodeURIComponent; abort-current-inline-script document.createElement String.fromCharCode
 
+egybest2.com,egybest4.com#$#abort-on-property-read adcashMacros; abort-on-property-write adcashMacros; abort-current-inline-script _0xbc31
```

Comment:
@ryanbr

I think you mean _0xbc13 instead of _0xbc31, at least this is what I'm getting matches for on my end.

Reply:
my err. fixed.